src/analysis/normed_space/operator_norm.lean

```diff
@@ -1,193 +1,190 @@
 /-
 Copyright (c) 2019 Jan-David Salchow. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Jan-David Salchow, Sébastien Gouëzel
+Authors: Jan-David Salchow, Sébastien Gouëzel, Jean Lo
 
 The space of bounded linear maps
 
 Define the set of bounded linear maps between normed spaces and show basic facts about it. In
 particular
 
-(*) define a set L(E,F) of bounded linear maps between k normed spaces,
-(*) show that L(E,F) is a vector subspace of E → F,
-(*) define the 'operator norm' on L(E,F) and show that it induces the structure of a normed space
-    on L(E,F).
+(*) show that bounded linear maps are a vector subspace of E → F,
+(*) define the operator norm and show that it induces the structure of a normed space
+    on bounded linear maps.
 -/
 import algebra.module
 import analysis.normed_space.bounded_linear_maps
+import topology.metric_space.lipschitz
 
-variables {k : Type*}
-variables {E : Type*} {F : Type*}
+variables (k : Type*) (E : Type*) (F : Type*) (G : Type*)
 
+variables [normed_field k]
+variables [normed_space k E] [normed_space k F] [normed_space k G]
 
--- Define the subspace of bounded linear maps.
-section bounded_linear_maps
+noncomputable theory
 set_option class.instance_max_depth 50
 
+def bounded_linear_map : subspace k (E → F) :=
+{ carrier := {f : E → F | is_bounded_linear_map k f},
+  zero := is_bounded_linear_map.zero,
+  add := λ _ _, is_bounded_linear_map.add,
+  smul := λ _ _, is_bounded_linear_map.smul _ }
 
-variables [hnfk : normed_field k] [normed_space k E] [normed_space k F]
-include hnfk
+variables {k E F G}
 
-def bounded_linear_maps : subspace k (E → F) :=
-{ carrier := {A : E → F | is_bounded_linear_map k A},
-  zero := is_bounded_linear_map.zero,
-  add := assume A B, is_bounded_linear_map.add,
-  smul := assume c A, is_bounded_linear_map.smul c }
+namespace bounded_linear_map
+
+notation E ` →L[`:25 k `] ` F := bounded_linear_map k E F
 
--- introduce the local notation L(E,F) for the set of bounded linear maps.
-local notation `L(` E `,` F `)` := @bounded_linear_maps k E F _ _ _
+/-- Construct bounded linear map from is_bounded_linear_map -/
+def from_is_bounded_linear_map {f : E → F} (hf : is_bounded_linear_map k f) :
```

Comment:
It would be better to call this `is_bounded_linear_map.to_bounded_linear_map` (outside of the `bounded_linear_map` namespace). In this way, if you have an assumption `h : is_bounded_linear_map f` somewhere, then `h.to_bounded_linear_map` gives direct access to the corresponding bounded linear map.

Reply:
effected in ef8e8e5c1338234f5511d9401d610a3dcde9e2af .